# message file for information perceived by the robot
# trial id
int64 id
# object size, 1: small, 2: big
int64 object_size
# action of human, 1: looking at robot face, 2: looking at coloured object, 3: looking away
int64 human_action
# human expression, 1: happy, 2:sad, 3: neutral
int64 human_expression
